app/helpers/application_helper.rb

```diff
@@ -165,17 +165,14 @@ def active_nav_li(link)
   end
 
   def show_time(length)
-    h = length / 60
-    min = length - h * 60
+    h, min = length.divmod(60)
```

Comment:
Maybe we should discuss what return value we expect if length is nil?

nil, "", "0 h 0m" ?

@differentreality